### PR TITLE
[Feature] Front-end comments

### PR DESCRIPTION
**Pull request by TomHeatwole**
Commenting through front-end files that I worked on to explain anything that may be confusing to someone picking up this project.

### CODE REVIEW

**Comment by Ghalko** [comment]
Looks good @TomHeatwole, keep going.

File: meetings/approvals/tests.py [hunk 1]
```diff
@@ -1,12 +0,0 @@
-from django.test import TestCase
-from unittest import skip
-
-
-class TestApprovals(TestCase):
```
Comment (Ghalko) [inline meetings/approvals/tests.py:5]: Why is this removed?

Comment (TomHeatwole) [inline meetings/approvals/tests.py:5]: I noticed that it's never referenced in the urls.py file and asked @Tav0 who said it could go.

Comment (Ghalko) [inline meetings/approvals/tests.py:5]: This is a place for future tests, please do not remove.

 @Tav0 It isn't referenced in the `urls.py` because it is never meant to be seen as a URL. It runs with `inv test_all` and won't be skipped once #67 is finished. The idea is that I outlined tests so that they could be filled in, not deleted.

Comment (TomHeatwole) [inline meetings/approvals/tests.py:5]: Oh my bad, I actually thought this was a different chunk of code that I removed. I'll put it back.
